### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by brettz9**
Builds on #245 .

- Linting: Check hidden files
- npm: Update devDeps (coveralls, eslint-plugin-node, mocha, nyc)
- Maintenance: Add `.ncurc.js` to prevent peerDep. updates on running local `npm-check-updates`

### CODE REVIEW

**Comment by brettz9** [comment]
Again, having a minor issue with the performance budget, so failing one build.

File: .ncurc.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+module.exports = {
+    // Whitelist all for checking besides `peer` which indicates
```
Comment (lo1tuma) [inline .ncurc.js:4]: I didn’t know about `npm-check-updates`. I always used `npx david`, but this tool looks more straight forward. Thanks 👍.

**Comment by brettz9** [comment]
Cool, thanks! I've rebased!

**Comment by brettz9** [comment]
Oh--would you like an update for eslint 7 too?

**Comment by brettz9** [comment]
To make it easier for Github Actions, and as many other projects are dropping such support, you might want to drop end-of-lifed Node 8 support and just go to Node 10 (and add 14). I can add that to this PR if you like...

**Comment by lo1tuma** [comment]
Sounds good to me. However if we drop node 8 we should also require eslint > 7 as a peer dependency right?
I would be happy to accept such a PR.

**Comment by brettz9** [comment]
No need to require it as a peer dependency as eslint < 7 still works on Node 10-14. But if you didn't want to add older eslint versions to the Github Actions matrix to ensure multiple eslint versions are still working, and you wanted to ensure no consumers could accidentally use the plugin without warning if using eslint < 7, then we could bump the peerDeps to eslint 7--however you like.

**Comment by lo1tuma** [comment]
You are absolutely right.
But if we are already going to make a breaking change (dropping node 8) we can also combine that with dropping explicit support for older ESLint version (even if we are still compatible with older versions). WDYT?

**Comment by brettz9** [comment]
Sure, SGTM. And as it is only a warning, people could technically use an earlier ESLint version if they wished. I'll add it now on top of the other fixes for ESLint just added.

**Comment by brettz9** [comment]
Added to #247.

**Pull request merged**